doc/source/examples/multiphysics/rising-bubble-VOF/rising-bubble-VOF.rst

```diff
@@ -0,0 +1,301 @@
+
+Two fluids are present in this simulation, hence in the ``physical 
+properties`` subsection, their physical properties should be defined:
+
+
+.. code-block:: text
+
+    #---------------------------------------------------
+    # Physical Properties
+    #---------------------------------------------------
+    subsection physical properties
+        set number of fluids         = 2
+        subsection fluid 0
+            set density              = 1000
+            set kinematic viscosity  = 0.01
+        end
+        subsection fluid 1
+            set density              = 100
+            set kinematic viscosity  = 0.01
+        end
+    end
+
+We define two fluids here simply by setting the number of fluids to be :math:`2`.
+In ``subsection fluid 0``, we set the density and the kinematic viscosity for the phase associated with a VOF indicator of 0. 
+Similar procedure is done for the phase associated with a VOF indicator of 1 in ``subsection fluid 1``.
+
+
+""""""""""""""""""""""""""""""""
+Surface Tension Force
+""""""""""""""""""""""""""""""""
+
+
+In Lethe, surface tension force (:math:`{\bf{F_{\sigma}}}`) is calculated using the following equation [1, 2]:
+
+    .. math::
+        {\bf{F_{\sigma}}} = 2 \sigma k {\bf{\psi}}
+
+where :math:`\sigma`, :math:`k` and :math:`\bf{\psi}` denote surface tension coefficient, filtered curvature and filtered phase fraction gradient.
+
+The following equations calculate the filtered phase fraction gradient and filtered curvature, respectively.
+
+    .. math:: 
+        \int_\Omega {\bf{v}} . {\bf{\psi}} + \eta_n \nabla {\bf{v}} . \nabla {\bf{\psi}} d\Omega = \int_\Omega {\bf{v}} . \nabla {\phi} d\Omega
+
+    .. math:: 
+        \int_\Omega v k + \eta_k \nabla v . \nabla k d\Omega = \int_\Omega \nabla v . \frac{\bf{\psi}}{|\bf{\psi}|} d\Omega
+
+where :math:`v`, :math:`\bf{\psi}`, :math:`\eta_n \geq 0`, :math:`\phi`, :math:`k`, and :math:`\eta_k \geq 0` denote a test function, filtered phase fraction gradient, phase fraction gradient filter value, phase fraction, filtered curvature, and curvature filter value, respectively.
+
+.. tip::
+
+  Phase fraction gradient filter value (:math:`\eta_n`) and curvature filter value (:math:`\eta_k`) must be small values larger than 0. We recommend the following procedure to choose a proper value for these parameters: 1, Enable ``output auxiliary fields`` to write filtered phase fraction gradient and filtered curvature fields 2, Choose a small value for :math:`\eta = h/10`, where :math:`h` is the smallest mesh size 3, Run the simulation and check whether the filtered phase fraction gradient and filtered curvature fields are smooth and without oscillation 4, If the filtered phase fraction gradient and filtered curvature fields show oscillations, increase the value :math:`\eta` to a larger value (:math:`\eta = h/5`, for example), and repeat this process until reaching smooth filtered phase fraction gradient and filtered curvature fields without oscillations.
```

Comment:
```suggestion
  Phase fraction gradient filter value (:math:`\eta_n`) and curvature filter value (:math:`\eta_k`) must be small values larger than 0. We recommend the following procedure to choose a proper value for these parameters:
  1. Enable ``output auxiliary fields`` to write filtered phase fraction gradient and filtered curvature fields
  2. Choose a small value for :math:`\eta = h/10`, where :math:`h` is the smallest mesh size
  3. Run the simulation and check whether the filtered phase fraction gradient and filtered curvature fields are smooth and without oscillation
  4. If the filtered phase fraction gradient and filtered curvature fields show oscillations, increase the value :math:`\eta` to a larger value (:math:`\eta = h/5`, for example), and repeat this process until reaching smooth filtered phase fraction gradient and filtered curvature fields without oscillations.
```

Reply:
If I write this tip in the suggested format, it will be printed out of the tip section.